reana_client/cli/ping.py

```diff
@@ -81,3 +81,38 @@ def version(ctx):  # noqa: D301
     \t $ reana-client version
     """
     click.echo(__version__)
+
+
+@configuration_group.command("workspaces")
+@click.pass_context
+@add_access_token_options
+@check_connection
+def workspaces(ctx, access_token):  # noqa: D301
+    """List of available workspaces.
```

Comment:
Very minor: I guess it supposed to be `List available workspaces.` here for consistency as defined in `docs/cmd_list.txt`

Reply:
BTW,  you agree that `workspaces` command should be in the same file as `ping` and `version` as a configurations command? Or is better to have it's own file and description in `docs/cmd_list.txt`?

Reply:
IMHO it makes sense to have `workspaces` as a configurations command. It does not fit to any other category of commands and to create a separate one just for this one command I think would be too much. @tiborsimko , WDYT?